gwcli/storage.py

```diff
@@ -882,53 +909,6 @@ def _update_pool(self):
             # update the pool commit numbers
             pool._calc_overcommit()
 
-    def ui_command_resize(self, size):
```

Comment:
Why remove all of these?

Reply:
Because disk name will now contain slashes:

```
o- disks ..................................................... [92G, Disks: 2]
  | o- rbd/disk_1 ............................................ [disk_1 (90G)]
  | o- rbd/disk_2 ............................................ [disk_2 (2G)]
```
we are no longer able to do `cd /disks/<disk_id>`, so I've moved all disk commands to the `/disks` level.

E.g., you will now have to do 
```
cd /disks
resize rbd/disk_1 30G
```

instead of 
```
cd /disks/rbd/disk_1
resize 30G
```

Reply:
Now that you bring this up, that was exactly the reason why the dots were used. Can we just add a new sub-level to the UI for the pool? or use the backstore name in the UI?

Reply:
I'll add a new sub-level for the pool.